gallery/src/components/NxSmallTag/NxSmallTagLayoutExample.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxH1, NxP, NxSmallTag } from '@sonatype/react-shared-components';
+
+function NxSmallTagLayoutExample() {
+  return (
+    <>
+      <NxH1>
+        <span>H1 Header / 32px / Bold</span>
+        <NxSmallTag>Pink</NxSmallTag>
```

Comment:
Remove whitespace here too
```suggestion
        <span>H1 Header / 32px / Bold</span><NxSmallTag>Pink</NxSmallTag>
```